[libremidi] Build failure on x64-linux

Package: libremidi:x64-linux@4.5.0#1

**Host Environment**

- Host: x64-linux
- Compiler: GNU 15.1.1
-    vcpkg-tool version: 2025-07-16-d6c019e723df46cb8c36360e4174b111455567d3
    vcpkg-scripts version: 5abae32a6c 2025-07-21 (76 minutes ago)

**To Reproduce**

`vcpkg install libremidi[*] libremidi[*]:x64-linux-dynamic`

**Failure logs**

```
-- Using cached jcelerier-libremidi-v4.5.0.tar.gz
-- Cleaning sources at /home/joao2/Documents/GitHub/vcpkg/buildtrees/libremidi/src/v4.5.0-c7b9464d29.clean. Use --editable to skip cleaning for the packages you specify.
-- Extracting source /home/joao2/Documents/GitHub/vcpkg/downloads/jcelerier-libremidi-v4.5.0.tar.gz
-- Using source at /home/joao2/Documents/GitHub/vcpkg/buildtrees/libremidi/src/v4.5.0-c7b9464d29.clean
-- Found external ninja('1.12.1').
-- Configuring x64-linux
-- Building x64-linux-dbg
CMake Error at scripts/cmake/vcpkg_execute_build_process.cmake:134 (message):
    Command failed: /usr/sbin/cmake --build . --config Debug --target install -- -v -j13
    Working Directory: /home/joao2/Documents/GitHub/vcpkg/buildtrees/libremidi/x64-linux-dbg
    See logs for more information:
      /home/joao2/Documents/GitHub/vcpkg/buildtrees/libremidi/install-x64-linux-dbg-out.log

Call Stack (most recent call first):
  installed/x64-linux/share/vcpkg-cmake/vcpkg_cmake_build.cmake:74 (vcpkg_execute_build_process)
  installed/x64-linux/share/vcpkg-cmake/vcpkg_cmake_install.cmake:16 (vcpkg_cmake_build)
  ports/libremidi/portfile.cmake:23 (vcpkg_cmake_install)
  scripts/ports.cmake:206 (include)



```

<details><summary>/home/joao2/Documents/GitHub/vcpkg/buildtrees/libremidi/install-x64-linux-dbg-out.log</summary>

```
Change Dir: '/home/joao2/Documents/GitHub/vcpkg/buildtrees/libremidi/x64-linux-dbg'

Run Build Command(s): /usr/sbin/ninja -v -v -j13 install
[1/13] /usr/sbin/c++ -DLIBREMIDI_ALSA -DLIBREMIDI_EXPORTS -DLIBREMIDI_HAS_UDEV -DLIBREMIDI_NO_BOOST -DLIBREMIDI_PIPEWIRE -I/home/joao2/Documents/GitHub/vcpkg/buildtrees/libremidi/x64-linux-dbg -I/home/joao2/Documents/GitHub/vcpkg/buildtrees/libremidi/src/v4.5.0-c7b9464d29.clean -I/home/joao2/Documents/GitHub/vcpkg/buildtrees/libremidi/src/v4.5.0-c7b9464d29.clean/include -I/home/joao2/Documents/GitHub/vcpkg/installed/x64-linux/include -isystem /usr/include/pipewire-0.3 -isystem /usr/include/spa-0.2 -fPIC -g -std=gnu++20 -fPIC -Wall -Wextra -MD -MT CMakeFiles/libremidi.dir/include/libremidi/backends/emscripten/midi_in.cpp.o -MF CMakeFiles/libremidi.dir/include/libremidi/backends/emscripten/midi_in.cpp.o.d -o CMakeFiles/libremidi.dir/include/libremidi/backends/emscripten/midi_in.cpp.o -c /home/joao2/Documents/GitHub/vcpkg/buildtrees/libremidi/src/v4.5.0-c7b9464d29.clean/include/libremidi/backends/emscripten/midi_in.cpp
[2/13] /usr/sbin/c++ -DLIBREMIDI_ALSA -DLIBREMIDI_EXPORTS -DLIBREMIDI_HAS_UDEV -DLIBREMIDI_NO_BOOST -DLIBREMIDI_PIPEWIRE -I/home/joao2/Documents/GitHub/vcpkg/buildtrees/libremidi/x64-linux-dbg -I/home/joao2/Documents/GitHub/vcpkg/buildtrees/libremidi/src/v4.5.0-c7b9464d29.clean -I/home/joao2/Documents/GitHub/vcpkg/buildtrees/libremidi/src/v4.5.0-c7b9464d29.clean/include -I/home/joao2/Documents/GitHub/vcpkg/installed/x64-linux/include -isystem /usr/include/pipewire-0.3 -isystem /usr/include/spa-0.2 -fPIC -g -std=gnu++20 -fPIC -Wall -Wextra -MD -MT CMakeFiles/libremidi.dir/include/libremidi/backends/emscripten/midi_out.cpp.o -MF CMakeFiles/libremidi.dir/include/libremidi/backends/emscripten/midi_out.cpp.o.d -o CMakeFiles/libremidi.dir/include/libremidi/backends/emscripten/midi_out.cpp.o -c /home/joao2/Documents/GitHub/vcpkg/buildtrees/libremidi/src/v4.5.0-c7b9464d29.clean/include/libremidi/backends/emscripten/midi_out.cpp
[3/13] /usr/sbin/c++ -DLIBREMIDI_ALSA -DLIBREMIDI_EXPORTS -DLIBREMIDI_HAS_UDEV -DLIBREMIDI_NO_BOOST -DLIBREMIDI_PIPEWIRE -I/home/joao2/Documents/GitHub/vcpkg/buildtrees/libremidi/x64-linux-dbg -I/home/joao2/Documents/GitHub/vcpkg/buildtrees/libremidi/src/v4.5.0-c7b9464d29.clean -I/home/joao2/Documents/GitHub/vcpkg/buildtrees/libremidi/src/v4.5.0-c7b9464d29.clean/include -I/home/joao2/Documents/GitHub/vcpkg/installed/x64-linux/include -isystem /usr/include/pipewire-0.3 -isystem /usr/include/spa-0.2 -fPIC -g -std=gnu++20 -fPIC -Wall -Wextra -MD -MT CMakeFiles/libremidi.dir/include/libremidi/backends/emscripten/observer.cpp.o -MF CMakeFiles/libremidi.dir/include/libremidi/backends/emscripten/observer.cpp.o.d -o CMakeFiles/libremidi.dir/include/libremidi/backends/emscripten/observer.cpp.o -c /home/joao2/Documents/GitHub/vcpkg/buildtrees/libremidi/src/v4.5.0-c7b9464d29.clean/include/libremidi/backends/emscripten/observer.cpp
[4/13] /usr/sbin/c++ -DLIBREMIDI_ALSA -DLIBREMIDI_EXPORTS -DLIBREMIDI_HAS_UDEV -DLIBREMIDI_NO_BOOST -DLIBREMIDI_PIPEWIRE -I/home/joao2/Documents/GitHub/vcpkg/buildtrees/libremidi/x64-linux-dbg -I/home/joao2/Documents/GitHub/vcpkg/buildtrees/libremidi/src/v4.5.0-c7b9464d29.clean -I/home/joao2/Documents/GitHub/vcpkg/buildtrees/libremidi/src/v4.5.0-c7b9464d29.clean/include -I/home/joao2/Documents/GitHub/vcpkg/installed/x64-linux/include -isystem /usr/include/pipewire-0.3 -isystem /usr/include/spa-0.2 -fPIC -g -std=gnu++20 -fPIC -Wall -Wextra -MD -MT CMakeFiles/libremidi.dir/include/libremidi/backends/emscripten/midi_access.cpp.o -MF CMakeFiles/libremidi.dir/include/libremidi/backends/emscripten/midi_access.cpp.o.d -o CMakeFiles/libremidi.dir/include/libremidi/backends/emscripten/midi_access.cpp.o -c /home/joao2/Documents/GitHub/vcpkg/buildtrees/libremidi/src/v4.5.0-c7b9464d29.clean/include/libremidi/backends/emscripten/midi_access.cpp
[5/13] /usr/sbin/c++ -DLIBREMIDI_ALSA -DLIBREMIDI_EXPORTS -DLIBREMIDI_HAS_UDEV -DLIBREMIDI_NO_BOOST -DLIBREMIDI_PIPEWIRE -I/home/joao2/Documents/GitHub/vcpkg/buildtrees/libremidi/x64-linux-dbg -I/home/joao2/Documents/GitHub/vcpkg/buildtrees/libremidi/src/v4.5.0-c7b9464d29.clean -I/home/joao2/Documents/GitHub/vcpkg/buildtrees/libremidi/src/v4.5.0-c7b9464d29.clean/include -I/home/joao2/Documents/GitHub/vcpkg/installed/x64-linux/include -isystem /usr/include/pipewire-0.3 -isystem /usr/include/spa-0.2 -fPIC -g -std=gnu++20 -fPIC -Wall -Wextra -MD -MT CMakeFiles/libremidi.dir/include/libremidi/libremidi.cpp.o -MF CMakeFiles/libremidi.dir/include/libremidi/libremidi.cpp.o.d -o CMakeFiles/libremidi.dir/include/libremidi/libremidi.cpp.o -c /home/joao2/Documents/GitHub/vcpkg/buildtrees/libremidi/src/v4.5.0-c7b9464d29.clean/include/libremidi/libremidi.cpp
FAILED: CMakeFiles/libremidi.dir/include/libremidi/libremidi.cpp.o 
/usr/sbin/c++ -DLIBREMIDI_ALSA -DLIBREMIDI_EXPORTS -DLIBREMIDI_HAS_UDEV -DLIBREMIDI_NO_BOOST -DLIBREMIDI_PIPEWIRE -I/home/joao2/Documents/GitHub/vcpkg/buildtrees/libremidi/x64-linux-dbg -I/home/joao2/Documents/GitHub/vcpkg/buildtrees/libremidi/src/v4.5.0-c7b9464d29.clean -I/home/joao2/Documents/GitHub/vcpkg/buildtrees/libremidi/src/v4.5.0-c7b9464d29.clean/include -I/home/joao2/Documents/GitHub/vcpkg/installed/x64-linux/include -isystem /usr/include/pipewire-0.3 -isystem /usr/include/spa-0.2 -fPIC -g -std=gnu++20 -fPIC -Wall -Wextra -MD -MT CMakeFiles/libremidi.dir/include/libremidi/libremidi.cpp.o -MF CMakeFiles/libremidi.dir/include/libremidi/libremidi.cpp.o.d -o CMakeFiles/libremidi.dir/include/libremidi/libremidi.cpp.o -c /home/joao2/Documents/GitHub/vcpkg/buildtrees/libremidi/src/v4.5.0-c7b9464d29.clean/include/libremidi/libremidi.cpp
In file included from /home/joao2/Documents/GitHub/vcpkg/buildtrees/libremidi/src/v4.5.0-c7b9464d29.clean/include/libremidi/backends/pipewire.hpp:7,
                 from /home/joao2/Documents/GitHub/vcpkg/buildtrees/libremidi/src/v4.5.0-c7b9464d29.clean/include/libremidi/backends.hpp:33,
                 from /home/joao2/Documents/GitHub/vcpkg/buildtrees/libremidi/src/v4.5.0-c7b9464d29.clean/include/libremidi/libremidi.cpp:5:
/home/joao2/Documents/GitHub/vcpkg/buildtrees/libremidi/src/v4.5.0-c7b9464d29.clean/include/libremidi/backends/pipewire/midi_out.hpp:6:10: fatal error: readerwriterqueue.h: No such file or directory
    6 | #include <readerwriterqueue.h>
      |          ^~~~~~~~~~~~~~~~~~~~~
compilation terminated.
[6/13] /usr/sbin/c++ -DLIBREMIDI_ALSA -DLIBREMIDI_EXPORTS -DLIBREMIDI_HAS_UDEV -DLIBREMIDI_NO_BOOST -DLIBREMIDI_PIPEWIRE -I/home/joao2/Documents/GitHub/vcpkg/buildtrees/libremidi/x64-linux-dbg -I/home/joao2/Documents/GitHub/vcpkg/buildtrees/libremidi/src/v4.5.0-c7b9464d29.clean -I/home/joao2/Documents/GitHub/vcpkg/buildtrees/libremidi/src/v4.5.0-c7b9464d29.clean/include -I/home/joao2/Documents/GitHub/vcpkg/installed/x64-linux/include -isystem /usr/include/pipewire-0.3 -isystem /usr/include/spa-0.2 -fPIC -g -std=gnu++20 -fPIC -Wall -Wextra -MD -MT CMakeFiles/libremidi.dir/include/libremidi/midi_out.cpp.o -MF CMakeFiles/libremidi.dir/include/libremidi/midi_out.cpp.o.d -o CMakeFiles/libremidi.dir/include/libremidi/midi_out.cpp.o -c /home/joao2/Documents/GitHub/vcpkg/buildtrees/libremidi/src/v4.5.0-c7b9464d29.clean/include/libremidi/midi_out.cpp
FAILED: CMakeFiles/libremidi.dir/include/libremidi/midi_out.cpp.o 
/usr/sbin/c++ -DLIBREMIDI_ALSA -DLIBREMIDI_EXPORTS -DLIBREMIDI_HAS_UDEV -DLIBREMIDI_NO_BOOST -DLIBREMIDI_PIPEWIRE -I/home/joao2/Documents/GitHub/vcpkg/buildtrees/libremidi/x64-linux-dbg -I/home/joao2/Documents/GitHub/vcpkg/buildtrees/libremidi/src/v4.5.0-c7b9464d29.clean -I/home/joao2/Documents/GitHub/vcpkg/buildtrees/libremidi/src/v4.5.0-c7b9464d29.clean/include -I/home/joao2/Documents/GitHub/vcpkg/installed/x64-linux/include -isystem /usr/include/pipewire-0.3 -isystem /usr/include/spa-0.2 -fPIC -g -std=gnu++20 -fPIC -Wall -Wextra -MD -MT CMakeFiles/libremidi.dir/include/libremidi/midi_out.cpp.o -MF CMakeFiles/libremidi.dir/include/libremidi/midi_out.cpp.o.d -o CMakeFiles/libremidi.dir/include/libremidi/midi_out.cpp.o -c /home/joao2/Documents/GitHub/vcpkg/buildtrees/libremidi/src/v4.5.0-c7b9464d29.clean/include/libremidi/midi_out.cpp
In file included from /home/joao2/Documents/GitHub/vcpkg/buildtrees/libremidi/src/v4.5.0-c7b9464d29.clean/include/libremidi/backends/pipewire.hpp:7,
                 from /home/joao2/Documents/GitHub/vcpkg/buildtrees/libremidi/src/v4.5.0-c7b9464d29.clean/include/libremidi/backends.hpp:33,
                 from /home/joao2/Documents/GitHub/vcpkg/buildtrees/libremidi/src/v4.5.0-c7b9464d29.clean/include/libremidi/midi_out.cpp:5:
/home/joao2/Documents/GitHub/vcpkg/buildtrees/libremidi/src/v4.5.0-c7b9464d29.clean/include/libremidi/backends/pipewire/midi_out.hpp:6:10: fatal error: readerwriterqueue.h: No such file or directory
    6 | #include <readerwriterqueue.h>
      |          ^~~~~~~~~~~~~~~~~~~~~
compilation terminated.
[7/13] /usr/sbin/c++ -DLIBREMIDI_ALSA -DLIBREMIDI_EXPORTS -DLIBREMIDI_HAS_UDEV -DLIBREMIDI_NO_BOOST -DLIBREMIDI_PIPEWIRE -I/home/joao2/Documents/GitHub/vcpkg/buildtrees/libremidi/x64-linux-dbg -I/home/joao2/Documents/GitHub/vcpkg/buildtrees/libremidi/src/v4.5.0-c7b9464d29.clean -I/home/joao2/Documents/GitHub/vcpkg/buildtrees/libremidi/src/v4.5.0-c7b9464d29.clean/include -I/home/joao2/Documents/GitHub/vcpkg/installed/x64-linux/include -isystem /usr/include/pipewire-0.3 -isystem /usr/include/spa-0.2 -fPIC -g -std=gnu++20 -fPIC -Wall -Wextra -MD -MT CMakeFiles/libremidi.dir/include/libremidi/client.cpp.o -MF CMakeFiles/libremidi.dir/include/libremidi/client.cpp.o.d -o CMakeFiles/libremidi.dir/include/libremidi/client.cpp.o -c /home/joao2/Documents/GitHub/vcpkg/buildtrees/libremidi/src/v4.5.0-c7b9464d29.clean/include/libremidi/client.cpp
FAILED: CMakeFiles/libremidi.dir/include/libremidi/client.cpp.o 
/usr/sbin/c++ -DLIBREMIDI_ALSA -DLIBREMIDI_EXPORTS -DLIBREMIDI_HAS_UDEV -DLIBREMIDI_NO_BOOST -DLIBREMIDI_PIPEWIRE -I/home/joao2/Documents/GitHub/vcpkg/buildtrees/libremidi/x64-linux-dbg -I/home/joao2/Documents/GitHub/vcpkg/buildtrees/libremidi/src/v4.5.0-c7b9464d29.clean -I/home/joao2/Documents/GitHub/vcpkg/buildtrees/libremidi/src/v4.5.0-c7b9464d29.clean/include -I/home/joao2/Documents/GitHub/vcpkg/installed/x64-linux/include -isystem /usr/include/pipewire-0.3 -isystem /usr/include/spa-0.2 -fPIC -g -std=gnu++20 -fPIC -Wall -Wextra -MD -MT CMakeFiles/libremidi.dir/include/libremidi/client.cpp.o -MF CMakeFiles/libremidi.dir/include/libremidi/client.cpp.o.d -o CMakeFiles/libremidi.dir/include/libremidi/client.cpp.o -c /home/joao2/Documents/GitHub/vcpkg/buildtrees/libremidi/src/v4.5.0-c7b9464d29.clean/include/libremidi/client.cpp
In file included from /home/joao2/Documents/GitHub/vcpkg/buildtrees/libremidi/src/v4.5.0-c7b9464d29.clean/include/libremidi/backends/pipewire.hpp:7,
                 from /home/joao2/Documents/GitHub/vcpkg/buildtrees/libremidi/src/v4.5.0-c7b9464d29.clean/include/libremidi/backends.hpp:33,
                 from /home/joao2/Documents/GitHub/vcpkg/buildtrees/libremidi/src/v4.5.0-c7b9464d29.clean/include/libremidi/client.cpp:4:
/home/joao2/Documents/GitHub/vcpkg/buildtrees/libremidi/src/v4.5.0-c7b9464d29.clean/include/libremidi/backends/pipewire/midi_out.hpp:6:10: fatal error: readerwriterqueue.h: No such file or directory
    6 | #include <readerwriterqueue.h>
      |          ^~~~~~~~~~~~~~~~~~~~~
compilation terminated.
[8/13] /usr/sbin/c++ -DLIBREMIDI_ALSA -DLIBREMIDI_EXPORTS -DLIBREMIDI_HAS_UDEV -DLIBREMIDI_NO_BOOST -DLIBREMIDI_PIPEWIRE -I/home/joao2/Documents/GitHub/vcpkg/buildtrees/libremidi/x64-linux-dbg -I/home/joao2/Documents/GitHub/vcpkg/buildtrees/libremidi/src/v4.5.0-c7b9464d29.clean -I/home/joao2/Documents/GitHub/vcpkg/buildtrees/libremidi/src/v4.5.0-c7b9464d29.clean/include -I/home/joao2/Documents/GitHub/vcpkg/installed/x64-linux/include -isystem /usr/include/pipewire-0.3 -isystem /usr/include/spa-0.2 -fPIC -g -std=gnu++20 -fPIC -Wall -Wextra -MD -MT CMakeFiles/libremidi.dir/include/libremidi/midi_in.cpp.o -MF CMakeFiles/libremidi.dir/include/libremidi/midi_in.cpp.o.d -o CMakeFiles/libremidi.dir/include/libremidi/midi_in.cpp.o -c /home/joao2/Documents/GitHub/vcpkg/buildtrees/libremidi/src/v4.5.0-c7b9464d29.clean/include/libremidi/midi_in.cpp
FAILED: CMakeFiles/libremidi.dir/include/libremidi/midi_in.cpp.o 
/usr/sbin/c++ -DLIBREMIDI_ALSA -DLIBREMIDI_EXPORTS -DLIBREMIDI_HAS_UDEV -DLIBREMIDI_NO_BOOST -DLIBREMIDI_PIPEWIRE -I/home/joao2/Documents/GitHub/vcpkg/buildtrees/libremidi/x64-linux-dbg -I/home/joao2/Documents/GitHub/vcpkg/buildtrees/libremidi/src/v4.5.0-c7b9464d29.clean -I/home/joao2/Documents/GitHub/vcpkg/buildtrees/libremidi/src/v4.5.0-c7b9464d29.clean/include -I/home/joao2/Documents/GitHub/vcpkg/installed/x64-linux/include -isystem /usr/include/pipewire-0.3 -isystem /usr/include/spa-0.2 -fPIC -g -std=gnu++20 -fPIC -Wall -Wextra -MD -MT CMakeFiles/libremidi.dir/include/libremidi/midi_in.cpp.o -MF CMakeFiles/libremidi.dir/include/libremidi/midi_in.cpp.o.d -o CMakeFiles/libremidi.dir/include/libremidi/midi_in.cpp.o -c /home/joao2/Documents/GitHub/vcpkg/buildtrees/libremidi/src/v4.5.0-c7b9464d29.clean/include/libremidi/midi_in.cpp
In file included from /home/joao2/Documents/GitHub/vcpkg/buildtrees/libremidi/src/v4.5.0-c7b9464d29.clean/include/libremidi/backends/pipewire.hpp:7,
                 from /home/joao2/Documents/GitHub/vcpkg/buildtrees/libremidi/src/v4.5.0-c7b9464d29.clean/include/libremidi/backends.hpp:33,
                 from /home/joao2/Documents/GitHub/vcpkg/buildtrees/libremidi/src/v4.5.0-c7b9464d29.clean/include/libremidi/midi_in.cpp:5:
/home/joao2/Documents/GitHub/vcpkg/buildtrees/libremidi/src/v4.5.0-c7b9464d29.clean/include/libremidi/backends/pipewire/midi_out.hpp:6:10: fatal error: readerwriterqueue.h: No such file or directory
    6 | #include <readerwriterqueue.h>
      |          ^~~~~~~~~~~~~~~~~~~~~
compilation terminated.
[9/13] /usr/sbin/c++ -DLIBREMIDI_ALSA -DLIBREMIDI_EXPORTS -DLIBREMIDI_HAS_UDEV -DLIBREMIDI_NO_BOOST -DLIBREMIDI_PIPEWIRE -I/home/joao2/Documents/GitHub/vcpkg/buildtrees/libremidi/x64-linux-dbg -I/home/joao2/Documents/GitHub/vcpkg/buildtrees/libremidi/src/v4.5.0-c7b9464d29.clean -I/home/joao2/Documents/GitHub/vcpkg/buildtrees/libremidi/src/v4.5.0-c7b9464d29.clean/include -I/home/joao2/Documents/GitHub/vcpkg/installed/x64-linux/include -isystem /usr/include/pipewire-0.3 -isystem /usr/include/spa-0.2 -fPIC -g -std=gnu++20 -fPIC -Wall -Wextra -MD -MT CMakeFiles/libremidi.dir/include/libremidi/observer.cpp.o -MF CMakeFiles/libremidi.dir/include/libremidi/observer.cpp.o.d -o CMakeFiles/libremidi.dir/include/libremidi/observer.cpp.o -c /home/joao2/Documents/GitHub/vcpkg/buildtrees/libremidi/src/v4.5.0-c7b9464d29.clean/include/libremidi/observer.cpp
FAILED: CMakeFiles/libremidi.dir/include/libremidi/observer.cpp.o 
/usr/sbin/c++ -DLIBREMIDI_ALSA -DLIBREMIDI_EXPORTS -DLIBREMIDI_HAS_UDEV -DLIBREMIDI_NO_BOOST -DLIBREMIDI_PIPEWIRE -I/home/joao2/Documents/GitHub/vcpkg/buildtrees/libremidi/x64-linux-dbg -I/home/joao2/Documents/GitHub/vcpkg/buildtrees/libremidi/src/v4.5.0-c7b9464d29.clean -I/home/joao2/Documents/GitHub/vcpkg/buildtrees/libremidi/src/v4.5.0-c7b9464d29.clean/include -I/home/joao2/Documents/GitHub/vcpkg/installed/x64-linux/include -isystem /usr/include/pipewire-0.3 -isystem /usr/include/spa-0.2 -fPIC -g -std=gnu++20 -fPIC -Wall -Wextra -MD -MT CMakeFiles/libremidi.dir/include/libremidi/observer.cpp.o -MF CMakeFiles/libremidi.dir/include/libremidi/observer.cpp.o.d -o CMakeFiles/libremidi.dir/include/libremidi/observer.cpp.o -c /home/joao2/Documents/GitHub/vcpkg/buildtrees/libremidi/src/v4.5.0-c7b9464d29.clean/include/libremidi/observer.cpp
In file included from /home/joao2/Documents/GitHub/vcpkg/buildtrees/libremidi/src/v4.5.0-c7b9464d29.clean/include/libremidi/backends/pipewire.hpp:7,
                 from /home/joao2/Documents/GitHub/vcpkg/buildtrees/libremidi/src/v4.5.0-c7b9464d29.clean/include/libremidi/backends.hpp:33,
                 from /home/joao2/Documents/GitHub/vcpkg/buildtrees/libremidi/src/v4.5.0-c7b9464d29.clean/include/libremidi/observer.cpp:5:
/home/joao2/Documents/GitHub/vcpkg/buildtrees/libremidi/src/v4.5.0-c7b9464d29.clean/include/libremidi/backends/pipewire/midi_out.hpp:6:10: fatal error: readerwriterqueue.h: No such file or directory
    6 | #include <readerwriterqueue.h>
      |          ^~~~~~~~~~~~~~~~~~~~~
compilation terminated.
[10/13] /usr/sbin/c++ -DLIBREMIDI_ALSA -DLIBREMIDI_EXPORTS -DLIBREMIDI_HAS_UDEV -DLIBREMIDI_NO_BOOST -DLIBREMIDI_PIPEWIRE -I/home/joao2/Documents/GitHub/vcpkg/buildtrees/libremidi/x64-linux-dbg -I/home/joao2/Documents/GitHub/vcpkg/buildtrees/libremidi/src/v4.5.0-c7b9464d29.clean -I/home/joao2/Documents/GitHub/vcpkg/buildtrees/libremidi/src/v4.5.0-c7b9464d29.clean/include -I/home/joao2/Documents/GitHub/vcpkg/installed/x64-linux/include -isystem /usr/include/pipewire-0.3 -isystem /usr/include/spa-0.2 -fPIC -g -std=gnu++20 -fPIC -Wall -Wextra -MD -MT CMakeFiles/libremidi.dir/include/libremidi/reader.cpp.o -MF CMakeFiles/libremidi.dir/include/libremidi/reader.cpp.o.d -o CMakeFiles/libremidi.dir/include/libremidi/reader.cpp.o -c /home/joao2/Documents/GitHub/vcpkg/buildtrees/libremidi/src/v4.5.0-c7b9464d29.clean/include/libremidi/reader.cpp
[11/13] /usr/sbin/c++ -DLIBREMIDI_ALSA -DLIBREMIDI_EXPORTS -DLIBREMIDI_HAS_UDEV -DLIBREMIDI_NO_BOOST -DLIBREMIDI_PIPEWIRE -I/home/joao2/Documents/GitHub/vcpkg/buildtrees/libremidi/x64-linux-dbg -I/home/joao2/Documents/GitHub/vcpkg/buildtrees/libremidi/src/v4.5.0-c7b9464d29.clean -I/home/joao2/Documents/GitHub/vcpkg/buildtrees/libremidi/src/v4.5.0-c7b9464d29.clean/include -I/home/joao2/Documents/GitHub/vcpkg/installed/x64-linux/include -isystem /usr/include/pipewire-0.3 -isystem /usr/include/spa-0.2 -fPIC -g -std=gnu++20 -fPIC -Wall -Wextra -MD -MT CMakeFiles/libremidi.dir/include/libremidi/writer.cpp.o -MF CMakeFiles/libremidi.dir/include/libremidi/writer.cpp.o.d -o CMakeFiles/libremidi.dir/include/libremidi/writer.cpp.o -c /home/joao2/Documents/GitHub/vcpkg/buildtrees/libremidi/src/v4.5.0-c7b9464d29.clean/include/libremidi/writer.cpp
ninja: build stopped: subcommand failed.
```
</details>

--- a/ports/libremidi/portfile.cmake
+++ b/ports/libremidi/portfile.cmake
@@ -19,9 +19,11 @@ vcpkg_cmake_configure(
         ${options}
         -DLIBREMIDI_NO_BOOST=ON
         -DLIBREMIDI_NO_JACK=ON
+        -DLIBREMIDI_NO_PIPEWIRE=ON
 )
 vcpkg_cmake_install()
-vcpkg_cmake_config_fixup(CONFIG_PATH lib/cmake/${PORT})
+vcpkg_cmake_config_fixup(CONFIG_PATH "lib/cmake/${PORT}")
 
 file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug/include")
+
 vcpkg_install_copyright(FILE_LIST "${SOURCE_PATH}/LICENSE.md")

--- a/ports/libremidi/vcpkg.json
+++ b/ports/libremidi/vcpkg.json
@@ -1,17 +1,13 @@
 {
   "name": "libremidi",
   "version": "4.5.0",
-  "port-version": 1,
+  "port-version": 2,
   "description": "A modern C++ MIDI real-time & file I/O library",
   "homepage": "https://github.com/jcelerier/libremidi",
   "license": "BSD-2-Clause",
   "dependencies": [
     {
       "name": "alsa",
-      "platform": "linux"
-    },
-    {
-      "name": "readerwriterqueue",
       "platform": "linux"
     },
     {

--- a/scripts/test_ports/vcpkg-ci-libremidi/portfile.cmake
+++ b/scripts/test_ports/vcpkg-ci-libremidi/portfile.cmake
@@ -1,0 +1,6 @@
+set(VCPKG_POLICY_EMPTY_PACKAGE enabled)
+
+vcpkg_cmake_configure(
+    SOURCE_PATH "${CURRENT_PORT_DIR}/project"
+)
+vcpkg_cmake_build()

--- a/scripts/test_ports/vcpkg-ci-libremidi/project/CMakeLists.txt
+++ b/scripts/test_ports/vcpkg-ci-libremidi/project/CMakeLists.txt
@@ -1,0 +1,8 @@
+cmake_minimum_required(VERSION 3.30)
+project(libremidi-test CXX)
+
+find_package(libremidi CONFIG REQUIRED)
+add_library(imported::libremidi ALIAS libremidi)
+
+add_executable(main main.cpp)
+target_link_libraries(main PRIVATE imported::libremidi)

--- a/scripts/test_ports/vcpkg-ci-libremidi/project/main.cpp
+++ b/scripts/test_ports/vcpkg-ci-libremidi/project/main.cpp
@@ -1,0 +1,12 @@
+#include <libremidi/libremidi.hpp>
+#include <libremidi/backends.hpp>
+#include <iostream>
+
+int main()
+{
+    std::cout << "Default midi2 API: " << libremidi::get_api_display_name(libremidi::midi2::default_api()) << std::endl;
+    libremidi::midi_any::for_all_backends([](auto& backend) {
+        std::cout << "- " << backend.display_name << std::endl;
+    });
+    return 0;
+}

--- a/scripts/test_ports/vcpkg-ci-libremidi/vcpkg.json
+++ b/scripts/test_ports/vcpkg-ci-libremidi/vcpkg.json
@@ -1,0 +1,12 @@
+{
+  "name": "vcpkg-ci-libremidi",
+  "version-string": "ci",
+  "description": "Validates libremidi",
+  "dependencies": [
+    "libremidi",
+    {
+      "name": "vcpkg-cmake",
+      "host": true
+    }
+  ]
+}

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -5286,7 +5286,7 @@
     },
     "libremidi": {
       "baseline": "4.5.0",
-      "port-version": 1
+      "port-version": 2
     },
     "libressl": {
       "baseline": "4.1.0",

--- a/versions/l-/libremidi.json
+++ b/versions/l-/libremidi.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "a5a10d57f2032120b80d4645366bc7290a4f149e",
+      "version": "4.5.0",
+      "port-version": 2
+    },
+    {
       "git-tree": "a9f7f74d1c8742a224f1bd79011ac5fee7a0247b",
       "version": "4.5.0",
       "port-version": 1

